_includes/templates/nav/about-us-media.html

```diff
@@ -0,0 +1,11 @@
+<a href="/the-bureau/">
+    <img src="../../static/img/nav-about-us-video.png"
+         alt="About Us video image" />
+    <h4 class="short-desc u-mt15">
+      The CFPB: Working for you.
+    </h4>
+</a>
+<div class="short-desc">
+    This short video covers what the CFPB is and
```

Comment:
This could be a better call to action, something like "Watch our short video to learn about the CFPB and how we go to  work for you."